src/mdbExtensionController.ts

```diff
@@ -210,6 +211,36 @@ export default class MDBExtensionController implements vscode.Disposable {
     }
   }
 
+  registerUriHandler = (): void => {
+    vscode.window.registerUriHandler({
+      handleUri: async (uri: vscode.Uri): Promise<void> => {
+        const command = uri.path.replace(/^\//, '');
```

Comment:
could it be worth dropping the need for one to specify `mdb.` to make the URLs a little nicer?

One potential risk of this approach: it does mean we need to be careful with ever renaming our command names as that'd break the URLs.

Reply:
We could - no strong feelings here, as I assumed those deeplink URLs will be generated and not something humans actually read too much into (e.g. the double encoding of the connection string already makes the URL quite obnoxious 😅). In this case, we'll probably need to document the need to drop the prefix when crafting the url.

Reply:
yeah that's fair. I don't have strong feelings about it